### DISCUSSION TITLE
When can Vercel enable `ap-southeast-2` for Posgres (Neon) storage solutions?

### DISCUSSION THREAD

**Discussion by qt-pixels** [vercel/community#5720] [Feature Question] [answered]
### Question

Given Neon have opened up `ap-southeast-2` from their end, I'm curious how long it would take for Vercel to allow us to select `ap-southeast-2` when configuring our Postgres service? 

[Neon's announcement](https://neon.tech/docs/changelog/2024-02-02-console)

**Comment by swarnava** [comment]
Let us check this and get back to you! :)

**Comment by swarnava** [comment, marked as answer]
This is available now!
